composer.json

```diff
@@ -24,16 +24,11 @@
         "symfony/monolog-bundle": "^2.3|^3.0",
         "symfony-cmf/routing-bundle": "^1.4|^2.0",
         "symfony-cmf/testing": "^2.0",
-        "twig/twig": "^1.18|^2.0",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
-        "sonata-project/admin-bundle": "^2.3.7|^3.0",
-        "sonata-project/block-bundle": "^2.2.8|^3.0",
-        "sonata-project/core-bundle": "^2.2.5|^3.0"
+        "twig/twig": "^1.18|^2.0"
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
     "suggest": {
-        "sonata-project/doctrine-phpcr-admin-bundle": "if you want to have editing capabilities (^1.1)",
```

Comment:
As two examples (seo, routing) are still merged with that suggestion, i think we should add it here and in the other. Lets have a little voting afterwards.

Even when i think those suggestions doen't really matter (and they are ugly white while an composer run), but from a marketing view, we should use every little chanel to show our ideas - even the really little ones.